Buildings/Resources/Images/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Staging/SetPoints/Subsequences/EfficiencyCondition_v2.svg

```diff
@@ -0,0 +1,965 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@milicag I changed this up so that it doesn't give the wrong impression that the circle-in-circle represents nested conditions. Let me know if the new diagram makes sense.
I am in favor of the single diagram since the stage type is continuously monitored and is not a one-time setting when the sequence is set-up. I will split it into two diagrams if you feel it's still not clear enough.

Reply:
@milicag Question 1

Reply:
Great, thanks, I'll take a look.

Reply:
I don't think I find this version any better, I think the previous version was better compared to this one. I would prefer if we were to stick to my initial recommendation. We need to reduce the possibility for user to misinterpret the diagram. What a two circles diagram would also help with is that the user always looks at the same type of a diagram, so no need for the user to interpret an illustration that looks like a new type of a diagram and in fact it is not. Two circles would also help in a way that you would not need the fist pseudo code line above each curved arrow. Please remember that not everyone is familiar with coding that uses the library. So the boiler types could be simply subtitles above each circle.